dotnet-desktop-guide/net/winforms/whats-new/net50.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: What's new in Windows Forms
-description: Learn about what's new in Windows Forms for .NET. Windows Forms. .NET provides new features and enhancements over .NET Framework.
+title: What's new in Windows Forms .NET 5
+description: Learn about what's new in Windows Forms for .NET 5. Windows Forms. .NET provides new features and enhancements over .NET Framework.
 ms.date: 11/05/2020
 ms.topic: conceptual
 ---
 
-# What's new (Windows Forms .NET)
+# What's new for .NET 5 (Windows Forms .NET)
```

Comment:
Why is this .NET 5 where everything else is .NET 6? Am I missing the purpose of the file?

Reply:
Never mind 😊 I see the file name now. You can disregard.